common/transform/src/main/java/ai/h2o/mojos/deploy/common/transform/MojoScorer.java

```diff
@@ -46,8 +43,8 @@
   // note the mojo pipeline need to be reloaded here as we have a constrain from java mojo
   // both SHAP values and predictions cannot be provided with the same pipeline
   // Link: https://github.com/h2oai/mojo2/blob/7a1ab76b09f056334842a5b442ff89859aabf518/doc/shap.md
-  private static final MojoPipeline pipelineShapley = loadMojoPipelineFromFile();
-
+  private static final MojoPipeline pipelineTransformedShapley = loadMojoPipelineFromFile();
```

Comment:
:scream: we load the mojo 3 times now? We shouldn't really be loading the mojo more than once. Like what happens if the user provides a 3GB mojo -- then each load == 36+ GB in memory * 3 = 108 GB just to have scorer for one mojo. Apologies, I should have caught this in the original pr, imho this really shouldn't happen. cc @mmalohlava wdyt? maybe I'm misunderstanding how mojo is loaded?

Reply:
yes, because the mojo pipeline/runtime still doesn't support this.

Reply:
also, if this really is the case, then we should be much more careful about this. MLops does not yet have resource control (i.e. being able to define the amount of cpus and memory assigned to a deployment) this could very easily break.

Reply:
cc @orendain what could be the impact of trying to do this in mlops? I am personally scared myself. How do you think kubernetes would respond in the instance that we try to deploy an extraordinarily large mojo?